src/main/java/core/basesyntax/serviceimpl/ReportGenerator.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.serviceimpl;
+
+import java.util.Map;
+
+public class ReportGenerator {
+    private static final String COLUMN_1 = "fruit";
+    private static final String COLUMN_2 = "quantity";
+    private static final String SEPARATOR = ",";
+
+    public String generateReport(Map<String, Integer> values) {
+        StringBuilder builder = new StringBuilder();
+        builder.append(COLUMN_1).append(SEPARATOR).append(COLUMN_2).append(System.lineSeparator());
```

Comment:
you append only constant values at line 12 here, it is more efficient to make a single constant from it and put it in the SB constructor instead
```suggestion
        StringBuilder builder = new StringBuilder(TITLE);
        for (...) {
```